azext-utils-api.d.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// For now this file needs to be copied/pasted into your repo if you want the types. Eventually we may put it somewhere more distributable.
+
+export interface AzureExtensionApi {
+    /**
+     * The API version for this extension. It should be versioned separately from the extension and ideally remains backwards compatible.
+     */
+    apiVersion: string;
+}
+
+export interface AzureExtensionApiProvider {
```

Comment:
Can we just get these declarations straight from the package? (Once we've actually merged and published)

Reply:
Oh, it's actually in `apiUtils` from `@microsoft/vscode-azureresources-api`